contents/blog/posthog-alternatives.md

```diff
@@ -0,0 +1,727 @@
+---
+title: "In-depth: PostHog alternatives, compared"
+date: 2023-05-25
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+author: ["andy-vandervell"]
+featuredImage: ../images/blog/prodhog.png
+featuredImageType: full
+category: General
+tags:
+  - Comparisons
+---
+
+## Amplitude
+
+- **Founded:** 2012
+- **Similar to:** Mixpanel, Heap
+- **Best known for:** Product analytics
+- **Useful for:** Product managers, growth and marketing teams
+
+### In their own words
+
+> "We are a digital analytics platform. We help every business optimize the business value of digital product innovation. We’re the competitive edge to not only survive—but thrive—in the new era."
+
+### Amplitude vs PostHog at a glance
```

Comment:
I think we could do more here to help buyers make a decision. It's nice to a have a good list of features, but I think often people dont know what they'll want. It would be nice to add a section on something like "Amplitude is recommended if your goal is X, and your role is Y, and you dont care about Z. PostHog is good for you if your goal is X, but you also care about Z. However, if you care about W or are trying to achieve Y, then Amplitude is recommended"

Basically what Im trying to say is lets make this more goal orientated and help users make a decision, rather than providing a list of features and leaving it up to them to parse.

----

Highly recommend [this talk](https://youtu.be/1DxQddmWqzQ?t=964) by April Dunford. I would say first 20 minutes are the most useful (a bit fluffy though), but if you have less time, I recommend watching from timestamp 16:00 for about 5 min.
TLDR:
* Buyers are overwhelmed by which decisions to make, since they're bombarded with information and it's difficult to know how to decide what's useful for them
* Most deals lose out to the status quo as a result.
* Help buyers make informed decisions by being the "informed sales people" that understands their needs

Reply:
Yeah, I like this idea. Will have a think about the best way / place to add something like this. It's basically something like "Choose Amplitude if..." etc. Maybe this can go in / after the Key differences section?

Reply:
Yeah that could work. Although Im undecided if it should at the beginning or at the end. I think at the beginning, as its helpful to keep in mind when reading the features list that comes after

Reply:
On reflection, I'm not adding this. I think it's best to make this page feel as "non-marketing" and neutral as possible. I've added an intro that's about as close to the line as I dare, though. I think we win by not trying spoon feed too much.